app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :correct_user, only: %i[edit update destroy]
+  before_action :set_commentable, only: %i[edit update create destroy]
+  before_action :set_comment, only: %i[edit update destroy]
+
+  def edit; end
+
+  def create
+    @comment = @commentable.comments.build(comment_params)
+
+    respond_to do |format|
+      if @comment.save
+        format.html { redirect_to polymorphic_path(@commentable), notice: t('controllers.common.notice_create', name: Comment.model_name.human) }
```

Comment:
```suggestion
        format.html { redirect_to @commentable, notice: t('controllers.common.notice_create', name: Comment.model_name.human) }
```

polymorphic_pathは省略することが多いかも